src/ActivitiesSDKAdapter.js

```diff
@@ -20,21 +21,54 @@ import logger from './logger';
  * @see {@link https://github.com/webex/component-adapter-interfaces/blob/master/src/ActivitiesAdapter.js#L6}
  */
 
+/**
+ * Extracts JSON card and files from server activity.
+ *
+ * @private
+ * @param {object} sdkActivity  SDK activity object
+ * @returns {Array} Array of attachments.
+ */
+function parseSDKAttachments(sdkActivity) {
+  const {attachments = []} = sdkActivity;
+
+  if (sdkActivity.attachments) {
+    attachments.concat(sdkActivity.attachments);
```

Comment:
won't this double the attachments extracted on line 32?

Reply:
Nice one, will fix.